packages/cli/src/lib/decorators/preconditions/trackable.ts

```diff
@@ -68,7 +68,7 @@ function getEventName(target: Command) {
   return target.id!.replace(/:/g, ' ');
 }
 
-function buildEvent(
+export function buildEvent(
```

Comment:
I would extract this method and `errorIdentifier` in another file (e.g. `src/lib/analytics/eventUtils.ts`)

Like this we would keep a good import tree (i.e. I do not need a precondition/decorator for a Oclif hook)